src/facebook_client.py

```diff
@@ -0,0 +1,128 @@
+import json
+
+from core_data_modules.data_models import validators
+from core_data_modules.logging import Logger
+import requests
+from core_data_modules.traced_data import TracedData, Metadata
+from core_data_modules.util import TimeUtils
+from dateutil.parser import isoparse
+
+log = Logger(__name__)
+
+_BASE_URL = "https://graph.facebook.com/v8.0"
+_MAX_LIMIT = 100
```

Comment:
add a comment on what this limit covers

Reply:
Added a comment, and renamed to _MAX_RESULTS_PER_PAGE, to help with clarity